### ISSUE TITLE
Julia specific explanations, instead of comparisons with R everywhere

### ISSUE THREAD

**Issue by smishr** [xKDR/Survey.jl#163]
User wrote in [discourse post](https://discourse.julialang.org/t/suggestions-for-the-design-of-survey-jl/86381/2):

"Hi! I’m currently not a Survey.jl user (don’t use R too), but when I opened documentation, I saw than it hard to understand exactly what package do without using R documentation. I think, it will be useful if you add documentation about package purpose and functionalities for only-Julia-users."

**Comment by iuliadmtru** [comment]
The main page of the documentation and the README don't contain comparisons with R anymore. Although they will become outdated once the singledesign branch is merged into main, the idea remains that we took this into consideration and it is not an issue anymore. I suggest closing this.
